.formatter.exs

```diff
@@ -1,6 +1,4 @@
+# Used by "mix format"
```

Comment:
I think this is rather obvious 😕 all changes in this file seem unnecessary though